Files/Filesystem/DriveItem.cs

```diff
@@ -74,7 +75,10 @@ public DriveType Type
             set
             {
                 type = value;
-                SetGlyph(type);
+                if (Icon == null)
+                {
+                    SetGlyph(type);
+                }
```

Comment:
@yaichenbaum

Reply:
For now, it should fallback to the previous SetGlyph behavior

Reply:
It doesn't look like the fallback is doing anything on 20H1.

Reply:
@yaichenbaum Can you elaborate because that could mean a number of things?

Reply:
Shouldn't I see the Segoe glyphs instead of the icons from File Explorer on 20H1?

Reply:
@yaichenbaum That's not included in this PR.

Reply:
Wasn't the previous SetGlyph behavior to use the segoe assets?

Reply:
@duke7553 When is it supposed to fallback to that?